helm_deploy/hmpps-template-kotlin/values.yaml

```diff
@@ -33,14 +32,8 @@ generic-service:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
 
   allowlist:
-    office: "217.33.148.210/32"
-    health-kick: "35.177.252.195/32"
```

Comment:
fyi, (i'm pretty sure that) we found that the `health-kick` ip address is not part of the "internal" group